include/expire.php

```diff
@@ -5,44 +5,18 @@
 function expire_run(&$argv, &$argc){
 	global $a;
 
-	require_once('include/datetime.php');
-	require_once('include/items.php');
-	require_once('include/Contact.php');
+	require_once 'include/datetime.php';
+	require_once 'include/items.php';
+	require_once 'include/Contact.php';
 
 	load_hooks();
 
-	if (($argc == 2) && ($argv[1] == 'delete')) {
-		logger('Delete expired items', LOGGER_DEBUG);
-		// physically remove anything that has been deleted for more than two months
-		$r = dba::p("SELECT `id` FROM `item` WHERE `deleted` AND `changed` < UTC_TIMESTAMP() - INTERVAL 60 DAY");
-		while ($row = dba::fetch($r)) {
-			dba::delete('item', array('id' => $row['id']));
-		}
-		dba::close($r);
+	$r = q("DELETE FROM `item` WHERE `deleted` = 1 AND `changed` < UTC_TIMESTAMP() - INTERVAL 60 DAY");
```

Comment:
I think this should use the new DB functions, and not the old one. But @annando should have a look as well.